modules/runner/src/integrationTest/java/org/apache/ignite/internal/runner/app/ItNoThreadsLeftTest.java

```diff
@@ -95,6 +98,24 @@ public void test(TestInfo testInfo) throws Exception {
         }
     }
 
+    private Ignite startNode(TestInfo testInfo) throws NodeStoppingException {
```

Comment:
Seems that such startThenInitAndSometimesAddToClusterNodeCollection method is widely spread. Let's add it to IgniteAbstractTest of similar, wdyt?

Reply:
All usages of this patter are slightly different among all tests, what interface would you propose for this method?

Reply:
It's up to you))